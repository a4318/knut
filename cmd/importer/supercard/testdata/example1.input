sep=;
Kontonummer;Kartennummer;Konto-/Karteninhaber;Einkaufsdatum;Buchungstext;Branche;Betrag;Originalwährung;Kurs;Währung;Belastung;Gutschrift;Buchung
1425 0000 0000;1111 2222 3333 4444;OWNER;09.05.2021;A CHE;Tankstelle;3.20;CHF; ;CHF;3.20; ;10.05.2021
1425 0000 0000;1111 2222 3333 4444;OWNER;09.05.2021;B CHE;Mitgliedschaft in Sportclubs;3.00;CHF; ;CHF;3.00; ;11.05.2021
1425 0000 0000;1111 2222 3333 4444;OWNER;11.05.2021;C CHE;Ärztliche Dienstleistungen;10.00;CHF; ;CHF;10.00; ;12.05.2021
1425 0000 0000;1111 2222 3333 4444;OWNER;13.05.2021;D CHE;Elektronikgeschäfte, Radio/TV;167.10;CHF; ;CHF;167.10; ;17.05.2021
1425 0000 0000;1111 2222 3333 4444;OWNER;14.05.2021;E CHE;Warenhaus;73.00;CHF; ;CHF;73.00; ;17.05.2021
1425 0000 0000;1111 2222 3333 4444;OWNER;14.05.2021;F CHE;Warenhaus;66.00;CHF; ;CHF;66.00; ;17.05.2021
1425 0000 0000;1111 2222 3333 4444;OWNER;10.06.2021;G CHE;Warenhaus;9.00;CHF; ;CHF; ;9.00;14.06.2021
