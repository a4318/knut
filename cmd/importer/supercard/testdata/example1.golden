2021-05-09 "A CHE Tankstelle"
Liabilities:CreditCard Expenses:TBD                  3.2 CHF

2021-05-09 "B CHE Mitgliedschaft in Sportclubs"
Liabilities:CreditCard Expenses:TBD                    3 CHF

2021-05-11 "C CHE Ärztliche Dienstleistungen"
Liabilities:CreditCard Expenses:TBD                   10 CHF

2021-05-13 "D CHE Elektronikgeschäfte, Radio/TV"
Liabilities:CreditCard Expenses:TBD                167.1 CHF

2021-05-14 "E CHE Warenhaus"
Liabilities:CreditCard Expenses:TBD                   73 CHF

2021-05-14 "F CHE Warenhaus"
Liabilities:CreditCard Expenses:TBD                   66 CHF

2021-06-10 "G CHE Warenhaus"
Expenses:TBD           Liabilities:CreditCard          9 CHF

